mui/mui.py

```diff
@@ -140,31 +149,35 @@ def solve(bv: BinaryView):
         )
 
         if dialog.exec() == QDialog.Accepted:
+            if bv.session_data.mui_client_stub == None:
+                subprocess.Popen([Path(__file__).resolve().parent.parent / "muicore_server"])
+                time.sleep(3)
+                client_stub = ManticoreUIStub(grpc.insecure_channel("localhost:50010"))
+                bv.session_data.mui_client_stub = client_stub
```

Comment:
note on implementation, which IMO is a bit hacky right now:
- mui_client_stub variable is None by default
- when attempting to Solve with Manticore for the first time, a subprocess server binary is spawned, there's a 3 second delay to ensure the server can start accepting connections, then the client connects to the server and the client stub is used from then on
- when closing the Binja window, any processes using the port 50010 (randomly-chosen hardcoded port for now) is killed

Reply:
> 3 second delay to ensure the server can start accepting connections

Is there a more well-supported way to do this sort of _retry_? What happens if it's not up in 3 seconds?

This link might be relevant? https://stackoverflow.com/questions/64227270/use-retrypolicy-with-python-grpc-client